source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/organization-product-definition/list.tsx

```diff
@@ -0,0 +1,58 @@
+import { compose } from 'recompose';
+
+import { query, buildOptions, ProductDefinitionResource } from '@data';
+
+import { IProvidedProps as IFilterProps } from '@data/containers/api/with-filtering';
+import { IPaginateProps } from '@data/containers/api/pagination';
+import { ISortProps } from '@data/containers/api/sorting';
+import { IProvidedProps as IOrgProps } from '@data/containers/with-current-organization';
+
+export interface IOwnProps {
+  productDefinitions: ProductDefinitionResource;
+  error?: any;
+}
+
+type IProps =
+& IFilterProps
+& IPaginateProps
+& IOrgProps
+& ISortProps;
+
+export function withNetwork<TWrappedProps>(options = {}) {
+  return WrappedComponent => {
+    function mapNetworkToProps(passedProps: TWrappedProps & IProps) {
+      const {
+        applyPagination, currentPageOffset, currentPageSize,
+        applyFilter, filters, sortProperty, currentOrganization,
+        applySort,
+      } = passedProps;
+
+      const requestOptions = buildOptions({
+        include: ['product-definition']
+      });
+
+      return {
+        cacheKey: [
+          sortProperty, filters,
+          currentPageOffset, currentPageSize
+        ],
+        organizationProductDefinitions: [
+          q => {
+            let builder = q.findRelatedRecords(currentOrganization,'organizationProductDefinitions');
```

Comment:
this file's name does not match its purpose --
 - the file name is `list`, which if following the pattern of the other files, this file should be making a request to the index route of the organization-product-definitions api
 - since this file is only getting the related records of the current organization (or maybe any organization?),
   this file should move under the organization resource container folder.
- I also don't know if it makes sense to paginate this relationship, but sorting and filtering could still be viable.

Reply:
I think I'm not using this file anymore. I added in my tests